packages/web-components/src/components/cbp-tooltip/cbp-tooltip.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, Prop, Element, Host, h } from '@stencil/core';
+import { setCSSProps } from '../../utils/utils';
+
+@Component({
+  tag: 'cbp-tooltip',
+  styleUrl: 'cbp-tooltip.scss'
+})
+export class CbpTooltip {
+
+  
+  @Element() host: HTMLElement;
+  /** When set, specifies that the tooltip is open */
+  @Prop({ reflect: true }) open: boolean = false;
+ 
+  //todo: update from alignment to placement
+  @Prop({ reflect: true}) alignment: "top-left" | "top-center" | "top-left" | "right-top" | "right-center" | "right-bottom" | "bottom-left" | "bottom-center" | "bottom-right" | "left-top" | "left-center" | "left-bottom";
+  
+  /** Specifies a unique `ID` for the tooltip, used to wire up the controls and accessibility features. */
+  @Prop() uid: string;
```

Comment:
this should have a default using createUUID() (?) function coming from utils. I'm not sure it needs to be a prop, as it's being implemented in the wrong place.

Reply:
updated to use createNamespaceKey as discussed